alerter.c

```diff
@@ -1,32 +1,37 @@
 #include <stdio.h>
 #include <assert.h>
 
-int alertFailureCount = 0;
-
-int networkAlertStub(float celcius) {
-    printf("ALERT: Temperature is %.1f celcius.\n", celcius);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+void formatPrintfInputColorMap (int majorNumberIndex, int minorNumberIndex, const char* recvMajorColor, const char* recvMinorColor)

```

Comment:
File name and the code within does not match !! looks like a naming typo...
Please correct.